toolchain/check/testdata/class/inheritance_access.carbon

```diff
@@ -178,16 +178,53 @@ class B {
   }
 
   fn SomeFunc[self: Self]() -> i32{
-    // CHECK:STDERR: fail_non_inherited_access.carbon:[[@LINE+6]]:12: ERROR: Cannot access protected member `INTERNAL_CONSTANT` of type `Internal`.
+    // CHECK:STDERR: fail_non_inherited_access.carbon:[[@LINE+7]]:12: ERROR: Cannot access protected member `INTERNAL_CONSTANT` of type `Internal`.
     // CHECK:STDERR:     return self.internal.INTERNAL_CONSTANT;
     // CHECK:STDERR:            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     // CHECK:STDERR: fail_non_inherited_access.carbon:[[@LINE-30]]:17: The protected member `INTERNAL_CONSTANT` is defined here.
     // CHECK:STDERR:   protected let INTERNAL_CONSTANT: i32 = 5;
     // CHECK:STDERR:                 ^~~~~~~~~~~~~~~~~
+    // CHECK:STDERR:
     return self.internal.INTERNAL_CONSTANT;
   }
 }
 
+// --- fail_compound_member_access.carbon
+
+library "[[@TEST_NAME]]";
+
+base class A {
+  private var x: i32;
+}
+
+class B {
```

Comment:
Should this have `extend base: A;`?

Given that it doesn't, it's a little surprising that the error is about `x` having the wrong access control, rather than about `A` not being a base class of `B` or something like that, but we don't need to fix that right now.